doc/api/timers.md

```diff
@@ -215,7 +215,7 @@ changes:
 
 Schedules repeated execution of `callback` every `delay` milliseconds.
 
-When `delay` is larger than `2147483647` or less than `1`, the `delay` will be
+When `delay` is larger than `2147483647` or less than `1` or `NaN`, the `delay` will be
```

Comment:
I don't know which one would be easier for the user to understand, not a number or `NaN`

Reply:
`NaN` feels more correct I think

Reply:
thanks @debadree25 would you mind starting the workflow for this PR?

Reply:
Hi yes have started the workflows